packages/teams-js/src/public/nestedAppAuth.ts

```diff
@@ -0,0 +1,20 @@
+import { ensureInitialized } from '../internal/internalAPIs';
+import { runtime } from './runtime';
+
+/**
+ * @beta
+ * Nested app auth capabilities
+ */
+export namespace nestedAppAuth {
+  /**
+   * Checks if MSAL-NAA channel recommended by the host
+   * @returns true if host is recommending NAA channel and false otherwise
+   *
+   * @throws Error if {@linkcode app.initialize} has not successfully completed
+   *
+   * @beta
+   */
+  export function isChannelRecommended(): boolean {
```

Comment:
Is this same as isSupported() function as other modules? If so, is there any reason we name it differently? 
(as far as I am aware, isSupported() is a well established function name for checking if a capability is supported)

Reply:
ok, it looks your PR on web hub sdk explains this in some details. Let me look at that first to see if It resolves my question. Thanks.

Reply:
No. It serves differently than isSupported() , please take a look at the reason in SDK PR because I'm not sure if we can explain it here due to some wording restrictions. Let me know if you have any concerns.

Reply:
As discussed offline, there is no more concern regarding the name.